Source/missiles.cpp

```diff
@@ -4670,14 +4670,15 @@ void MI_Manashield(int i)
 				missile[i]._miDelFlag = TRUE;
 				if (plr[id]._pHitPoints < 0)
 					SetPlayerHitPoints(id, 0);
-				if (!(plr[id]._pHitPoints >> 6) && id == myplr) {
+				if ((plr[id]._pHitPoints >> 6) == 0 && id == myplr) {
 					SyncPlrKill(id, missile[i]._miVar8);
 				}
 			}
 		}
 
 #ifndef HELLFIRE
-		if (id == myplr && !plr[id]._pHitPoints && !missile[i]._miVar1 && plr[id]._pmode != PM_DEATH) {
+	if (id == myplr && plr[id]._pHitPoints == 0 && missile[i]._miVar1 == 0 && plr[id]._pmode != PM_DEATH)
+			{

```

Comment:
What are you doing down here?

Reply:
hitpoints and miVar1 are ints so I'm doing == 0 comparison on them instead of !

Reply:
ah this crap moved

Reply:
fixed